src/main/java/com/example/demo/Email.java

```diff
@@ -0,0 +1,102 @@
+package com.example.demo;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.springframework.stereotype.Component;
+
+import java.io.Serializable;
+import java.sql.SQLOutput;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Scanner;
+
+
+@Component
+public class Email implements Serializable {
+
+
+    private String recipientName=null;
+    private String emailSubject=null;
+    private String emailBody=null;
+    private Date deliveryDate=null;
+    private Boolean isSent=false;
+
+
+    protected void inputData(){
+        
+        Scanner in=new Scanner(System.in);
```

Comment:
This class looks like simple data structure and it shouldn't depend on console. I suggest you to move it from here into component(managed by spring) which is responsible for user interface